MobileCenter/MobileCenter/Internals/AppDelegate/MSAppDelegateForwarder.m

```diff
@@ -518,10 +534,14 @@ + (void)flushTraceBuffer {
   // Only trace once.
   static dispatch_once_t traceOnceToken;
   dispatch_once(&traceOnceToken, ^{
```

Comment:
Do we still need the `dispatch_once` now that everything is synchronized?

Reply:
Good point. Let's talk offline.